tests/equilibria/test_flux_surfaces.py

```diff
@@ -40,6 +41,44 @@
 TEST_PATH = get_bluemira_path("equilibria/test_data", subfolder="tests")
 
 
+class TestFluxSurface:
+    def setup_class(self):
+        eq_name = "DN-DEMO_eqref.json"
+        filename = os.sep.join([TEST_PATH, eq_name])
+        self.eq = Equilibrium.from_eqdsk(filename)
+
+    def test_calculate_poloidal_angle(self):
```

Comment:
Follow up on the function name change

```suggestion
    def test_poloidal_angle(self):
```